src/dra/5D6C4.c

```diff
@@ -798,282 +754,268 @@ void func_800FF7B8(s32 arg0) {
-                D_80097C18 = 0x39;
+                // And 1 of everything wearable
+                for (i = 0; i < 90; i++) {
+                    g_Status.equipBodyCount[i] = 1;
+                }
+                g_Status.equipment[0] = 0x13; // Short Sword
+                g_Status.equipment[1] = 5;    // Leather Shield
+                g_Status.equipment[2] = 0x1A; // No headgear
+                g_Status.equipment[3] = 2;    // Hide cuirass
+                g_Status.equipment[4] = 0x30; // No cape
+                g_Status.equipment[5] = 0x39; // No misc
+                g_Status.equipment[6] = 0x39; // No misc
                 g_Status.timerHours = 0;
                 g_Status.timerMinutes = 0;
                 g_Status.timerSeconds = 0;
                 g_Status.timerFrames = 0;
-                subWeapon.subWeapon = 0;
+                g_Status.subWeapon = 0;
                 g_Status.relics[10] = 3;
-                (&g_Status.relics[10])[1] = 3;
-                D_80097973 = 3;
-                *D_80097964 = 3;
-                D_80097965 = 3;
-                D_80097968 = 3;
-                D_80097969 = 3;
-                D_8009796A = 3;
-                D_8009796B = 3;
-                D_80097970 = 3;
-                D_80097971 = 3;
-                func_800FD874(0x6F, 0);
-                func_800FD874(0x70, 0);
-                func_800FD874(0x71, 0);
-                func_800FD874(0x62, 0);
-                func_800FD874(0x80, 0);
-                func_800FD874(0x64, 0);
-                func_800FD874(6, 0);
-                func_800FD874(7, 0);
-                func_800FD874(0x12, 0);
-                func_800FD874(0x17, 0);
-                func_800FD874(0x55, 0);
-                func_800FD874(0x58, 0);
-                func_800FD874(1, 2);
-                func_800FD874(3, 2);
-                func_800FD874(4, 2);
-                func_800FD874(5, 2);
-                func_800FD874(6, 2);
-                func_800FD874(7, 2);
-                func_800FD874(0xA, 2);
-                func_800FD874(0xD, 2);
-                func_800FD874(0x1F, 1);
-                func_800FD874(0x21, 1);
-                func_800FD874(0x23, 1);
-                func_800FD874(0x31, 3);
-                func_800FD874(0x33, 3);
-                func_800FD874(0x35, 3);
-                func_800FD874(0x32, 3);
-                func_800FD874(0x52, 4);
-                func_800FD874(0x4F, 4);
-
-                for (i = 0; i < 0x50; i++) {
-                    func_800FD874(0x9F, 0);
+                g_Status.relics[11] = 3;
+                g_Status.relics[15] = 3;
+                g_Status.relics[0] = 3;
+                g_Status.relics[1] = 3;
+                g_Status.relics[4] = 3;
+                g_Status.relics[5] = 3;
+                g_Status.relics[6] = 3;
+                g_Status.relics[7] = 3;
+                g_Status.relics[12] = 3;
+                g_Status.relics[13] = 3;
+                AddToInventory(0x6F, 0); // Firebrand
```

Comment:
This function has so many item IDs I'm wondering if it's a good time to create an enum?

Reply:
Possibly, depending on how many places reference item IDs. Implementing that and replacing all the existing uses of item ID's sounds like a full separate PR worth of work though.

Maybe we should make a Wiki page of "Things someone should look into that would help with the decompilation but are not actually decompiling assembly" - including constructing enums like this one. We track all sorts of progress on decompiled code/functions, but these other side tasks that don't increase percentage tend to be forgotten. Determining the purpose of D_80XXXX variables and giving them useful names could be another task of that type.